public/docs/advanced/commands.md

```diff
@@ -1,129 +1,215 @@
 # Command reference
 
+This section will describe every command available in the Truffle application.
+
+
 ## Usage
 
-```none
-$ truffle [command] [options]
+All commands are in the following form:
+
+```shell
+truffle [command] [options]
 ```
 
+No arguments given is equivalent to `truffle help`, which will display a list of all commands and then exit.
```

Comment:
I got confused on this line (tried to type my confusion but figured it wasn't worth the long stream of consciousness). I'd reword this to say 
> Passing no arguments is equivalent to `truffle help`

or

> If no arguments are given it's equivalent to typing `truffle help`. 
etc.